app/views/conversations/show.html.erb

```diff
@@ -1,12 +1,33 @@
+<h1> Conversation <%= @conversation.id %> </h1>
+
 <p>
   <strong>Company representative:</strong>
-  <%= @conversation.company_representative_id %>
+  <%= User.all.find(@conversation.company_representative_id).username.capitalize() %>
```

Comment:
`.all` is implied, so it's not necessary.

```suggestion
  <%= User.find(@conversation.company_representative_id).username.capitalize() %>
```

I might also make this into an instance variable in the conversations#show controller.